### ISSUE TITLE
Problem adding a new custom field extension

### ISSUE THREAD

**Issue by llxx-lord-xxll** [z-song/laravel-admin#1962]
- Laravel Version: 5.5.*
- PHP Version: 7.1.1
- Laravel-admin: 1.5.x-dev

### Description:
I want to add [ACEEditor](https://ace.c9.io/) as a new form field. I followed the structure how i added ckEditor with it. But i think if the class name of the Extension Field Class  is different from "CKEditor" is not loading in Form::extend function in bootstrap.php. 

### Steps To Reproduce:

I tested it by just copying the "CKEditor" file to the same directory and saved it to "CKEditors", then also changed the classname according to the filename. Then extended the form field from bootstrap.php. But it seemed not working. But when i'm using only CKEditor class and then it works as extending name "aceditor"

![image](https://user-images.githubusercontent.com/6489365/39095147-bf6fed50-465d-11e8-9981-5cd905dcae8f.png)
Renamed File which is not working, doesn't even call the render function[Tried by vardump random text here]

![image](https://user-images.githubusercontent.com/6489365/39095148-ca2e2e28-465d-11e8-8248-1c75d0c5bb72.png)
bootstrap.php

![image](https://user-images.githubusercontent.com/6489365/39095162-0c3e8f4c-465e-11e8-85f4-44a9a972e667.png)
Original CKEditor which works

![image](https://user-images.githubusercontent.com/6489365/39095188-42fe2628-465e-11e8-8253-c1706937ee6f.png)
The form building 

![image](https://user-images.githubusercontent.com/6489365/39095214-97223d3e-465e-11e8-99b7-d0b063f61cff.png)
The aceditor class with which i tried first, when i saw that it doesn't even call the render function, then i tried for those testing.

**Comment by z-song** [comment]
How do you register the `ACEditor` extension?

**Comment by z-song** [comment]
Just follow the same way, no anything else.

I saw that you have defined four editor classes. Check if you have confused them

**Comment by llxx-lord-xxll** [comment]
I did exactly the same. I created them just to test and make sure if my guess is true or not.

What i did to test is, i copied the same class CKEditor.php which works fine, i copied it and renamed to "CKEditors.php" and also renamed the classname in the file "CKEditors",
Then tried to point the \Encore\Admin\Form::extend('aceditor', \App\Admin\Extension\Form\CKEditors::class);   in bootstrap.php  and it did not work :( 

I thought it is about artisan cache, or something else. So i cleared the caches also. Relaunched the artisan serve, nothing helps
